setup.cfg

```diff
@@ -59,7 +61,7 @@ dev =
   keras-autodoc ==0.6.0
   mkdocs >=1.1.2,<2
   mkdocs-material >=7.0.6,<8
-  mike >=0.5.5,<1
+  mike >=1.0
```

Comment:
this is updated according to a comment left:
```
 # Below version is needed to work around a clash with ruamel-yaml. When v1.0 is released this can be removed.
pip install git+git://github.com/jimporter/mike.git@2961733cbfef132e9d63752307443981ad02d1fc --upgrade
```
the issue was that `mkdocs.yml` couldnt be declaring the same keyword multiple times. From 1.0 it seems to be working.